htsget.md

```diff
@@ -206,6 +207,20 @@ The server SHOULD respond with an `InvalidInput` error if any other htsget query
 </td></tr>
 <tr markdown="block"><td>
 
+`encryptionScheme`
+_optional_
+</td><td>
+
+Request data encrypted with a particular encryption scheme.
+
+
+
+If `encryptionScheme` is specified a 4 letter code for the encryption standard MUST be returned. When `c4gh` is specified, Crypt4GH payload for a particular object will be returned (if available).
+
+The server SHOULD reply with a `NotFound` error if the requested reference does not exist.
```

Comment:
This doesn't make sense and appears to have been copy/pasted from the `referenceName` section. Do you mean if the server does not recognise a requested encryption code?

Reply:
Good catch, I meant (encrypted) object Id, fixed in https://github.com/samtools/hts-specs/pull/808/commits/4e6fa1a36658d81bbf16c86867ca15f7cccfcf28

Reply:
If this is about the resource's `<id>`, i.e., what forms the bulk of the path part of the URL, then it should be in the `URL parameters` section (in fact, it seems this has always been missing there!). That would be a separate PR.

If this is about the 4-letter code designating the encryption method, then it should be here and use the same `code` terminology.